test/fuzz/test_runner.py

```diff
@@ -283,12 +290,19 @@ def job(command, t, t_env):
         futures.append(fuzz_pool.submit(job, command, target, t_env))
 
     for future in as_completed(futures):
-        future.result()
+        target, output = future.result()
+        if using_libfuzzer:
+            done_stat = [l for l in output.splitlines() if "DONE" in l]
+            assert len(done_stat) == 1
+            num_targets_completed = log_fuzz_progress(num_targets_completed, len(test_list), done_stat, target, target_max_len)
```

Comment:
I think it should be this:
```suggestion
            num_targets_completed = log_fuzz_progress(num_targets_completed, len(targets), done_stat, target, target_max_len)
```

My preference would probably change the argument `targets` to `test_list` (and also update the name in args.generate) because this is more consistent with the other modes. However I understand if this is out of scope.

Reply:
I actually had to update this since `test_list` wasnt defined in this function, so I just got the `num_of_targets` at the top and used it here [3af8f62](https://github.com/bitcoin/bitcoin/pull/32354/commits/3af8f625fa1087fc659f5cbf0c5f568f15d89ee1)

lemme know if that looks good!